lib/remote_coffee_slack/session_notifier.rb

```diff
@@ -28,9 +28,7 @@ def next_session_message
     end
 
     def coffee_mates_formated
-      coffee_mates.map do |mates|
-        "<#{mates.first}> and <#{mates.last}>\n"
-      end.join
+      coffee_mates.flatten.join(', ')
```

Comment:
Do you mind adding a test for this? 😄

Reply:
Hey @Amosgito Thanks for your contribution, recently I have updated the logic of this method to support also the DM notifications implemented on #14 

https://github.com/Calzada-Code/remote-coffee-slack/blob/450231eb8fd42fd3adade77cb3237917c9489000/lib/remote_coffee_slack/session_notifier.rb#L30-L33

As you can see, now I'm including the `@` in this method.

I have in mind the following suggestions:

- Could you create a workflow to show `@user1 and @user2` when the group is of 2 users, and `@user1, @user2, and @user3` when the group has more than 2 users? 
- Could you consider include a test for this?
- Please don't forget to include the `@` now that the coffee_mates array only include the user ids without the `@`

Thank for showing interest I'll be very happy to review the changes that you will send. Also if you have ideas about how to improve this gem please feel you free to open an issue with suggestions 😄